minichlink/minichlink.c

```diff
@@ -1464,7 +1880,11 @@ static int DefaultWriteWord( void * dev, uint32_t address_to_write, uint32_t dat
 		MCF.WriteReg32( dev, DMDATA1, address_to_write );
 		MCF.WriteReg32( dev, DMDATA0, data );
 
-		if( did_disable_req )
+		if( iss->target_chip_type == CHIP_CH58x || iss->target_chip_type == CHIP_CH585 )
+		{
+			ret |= MCF.WriteReg32( dev, DMCOMMAND, 0x00240000 ); // Execute.
```

Comment:
Are you sure about this?  It looks like ch58x should allow for autoexecute. Why would you special case this?